app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/ImportFilesScreen.kt

```diff
@@ -139,18 +187,40 @@ private fun ImportFilesScreen(
                 TransferTypeButtons(selectedTransferType)
                 ImportFilesTitle(Modifier.padding(vertical = Margin.Medium), titleRes = R.string.advancedSettingsTitle)
                 TransferAdvancedSettings(
-                    states = {
-                        listOf(
-                            ValidityPeriodOption.THIRTY,
-                            DownloadLimitOption.ONE,
-                            PasswordTransferOption.NONE,
-                            EmailLanguageOption.GERMAN,
-                        )
-                    },
-                    onClick = {},
+                    modifier = Modifier,
```

Comment:
Since you're not passing any Modifier, is that necessary ?

Reply:
of course not, it's oxbow code, good catch